invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -0,0 +1,29 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import { Modal } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { AccessRequestsTabContent } from "./AccessRequestsTabContent";
+
+export class AccessRequestsTab extends Component {
```

Comment:
this component is not needed, it does not provide any functionality, but increases complexity